docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,123 @@
+# Adding a New Metrics Provider for Placeholder Endpoint
+
+In this guide, we will create a placeholder provider.
+The following steps can be a starting point to create your own custom provider:
+
+1. Fork the [Keptn repo](https://github.com/keptn/lifecycle-toolkit)
```

Comment:
```suggestion
1. Fork and clone the [Keptn repo](https://github.com/keptn/lifecycle-toolkit)
```
Add a reference to https://keptn.sh/stable/docs/contribute/general/git/fork-clone/ -- so something like "See <link> for details" or "more information" or whatever.

Reply:
Alright